govcd/nsxt_edgegateway.go

```diff
@@ -87,6 +101,19 @@ func (vdc *Vdc) GetNsxtEdgeGatewayByName(name string) (*NsxtEdgeGateway, error)
 	return returnSingleNsxtEdgeGateway(name, allEdges)
 }
 
+// GetNsxtEdgeGatewayByName allows to retrieve NSX-T edge gateway by Name for specific VDC Group
+func (vdcGroup *VdcGroup) GetNsxtEdgeGatewayByName(name string) (*NsxtEdgeGateway, error) {
+	queryParameters := url.Values{}
+	queryParameters.Add("filter", "name=="+name)
```

Comment:
does the query will work if name is empty?

Reply:
Added validation to avoid any possible problems. `org.GetAllNsxtEdgeGateways` does filtering.